### PR TITLE
Adding the changes for the right version of the motech 0.27.6

### PR DESCRIPTION
**Pull request by rubenmartin**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by kkaczmarczyk** [comment]
test this please

File: pom.xml [hunk 1]
```diff
@@ -1,69 +1,71 @@
...
 
-    <parent>
-        <groupId>org.motechproject</groupId>
-        <artifactId>motech</artifactId>
-        <version>0.27.5</version>
-    </parent>
+	<parent>
+		<groupId>org.motechproject</groupId>
+		<artifactId>motech</artifactId>
+		<version>0.27.5</version>
```
Comment (pmuchowski) [inline pom.xml]: @rubenmartin you need to change the version here, I think you can use ${motech.version}

Comment (rubenmartin) [inline pom.xml]: I have added and I got this message:

[ERROR]   The project org.motechproject:motech-ui-testing:0.27.1-SNAPSHOT (C:\Users\User_1\Dropbox\sandbox\motech\ruben\UITestFramework\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Failure to find org.motechproject:motech:pom:${motech.version} in http://nexus.motechproject.org/content/repositories/public was cached in the local repository, resolution will not be reattempted until the update interval of motech-repo has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 34, column 10 -> [Help 2]
org.apache.maven.model.resolution.UnresolvableModelException: Failure to find org.motechproject:motech:pom:${motech.version} in http://nexus.motechproject.org/content/repositories/public was cached in the local repository, resolution will not be reattempted until the update interval of motech-repo has elapsed or updates are forced

I have all the modules , motech in the same level so all are in the same folder and the error still is there (http://goo.gl/L8I39H)

I would love to use the dynamic reference , perhaps is needed another change in the motech pom , is it not working right now.

Just the 0.27.6 works, so it seems something on configuration ?

Comment (pmuchowski) [inline pom.xml]: 0.27.6 is fine, I wasn't sure if ${motech.version} will work

**Comment by pmuchowski** [comment]
test this please

**Comment by pmuchowski** [comment]
test this please

**Comment by pmuchowski** [comment]
test this please

**Comment by pmuchowski** [comment]
test this please

**Pull request closed without merging**